### PR TITLE
List resources used by DS, and say that none of them are 3rd party.

### PR DESCRIPTION
**Pull request by danmelamed**

### CODE REVIEW

**Comment by lhoang29** [comment]
Can you put these comments below the deploy button instead (similar to other template repos)? Keeping the button on top makes it always immediately visible for use (although in an unrelated discussion, I think we should direct users to the RC for deployment so we can keep track of emails).

**Comment by lhoang29** [comment]
Also, since we will have these footnotes here, how about we remove the duplicate texts from the RC? We could include a link pointing here so users can view the disclaimer if desired. The UI of the RC will also be simple as a result, at the moment I'm afraid it's overcrowded with text.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # ds-provisioning
 
+A default instance of the Decision Service currently includes the following Azure resources:
+* storage account
+* clasic cloud service with standard D5 worker role
```
Comment (eisber) [inline README.md]: typo classic

**Comment by danmelamed** [comment]
I wanted to put the "manifest" above the button, to ensure that users see it before they see the click-thru license.  The point is to reduce the chances that they will be scared off.

I agree, however, that we should minimize duplication of functionality.  To that end, I propose that we have the deploy button (and the manifest) only on our website, and not on github.  The github repo can simply point/link to the website.  Would that work?

Re making the website front page less crowded, I agree, and I propose that the bottom half (with the virtuous cycle graphic) be moved to a separate page or simply removed.  Although it is eye-catching, that info is available in other places already.

Typo fixed.

**Comment by lhoang29** [comment]
Redirecting from github to RC seems ok to me, especially given we want to track usage. Re: UI, I can actually create a dialog displaying the manifest so we can solve both problems.

**Comment by lhoang29** [comment]
oops, accidentally merged. Anyway, we can strip off the manifest on this page and instead just add documentation to describe what the project is about.

**Pull request merged**